vexim/config/tests/functionsValidatePasswordTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+	
+class FunctionValidatePasswordTest extends PHPUnit_Framework_TestCase {
+		
+	public function setUp() {
+		require_once(__DIR__ . '/../functions.php');
+	}
+
+        public function testSendEmptyPasswords()
+        {
+            $this->assertFalse(validate_password("", ""));
+        }
+
+        public function testSendSamePasswords()
+        {
+            $this->assertTrue(validate_password("password", "password"));
+        }
+
+        public function testSendDifferentPasswords()
+        {
+            $this->assertFalse(validate_password("password", "pass"));
+        }
+
+        public function testSendNullValues()
+        {
+            $this->assertFalse(validate_password(null, null));
+        }
+
+        public function testSendBoolValues()
+        {
+            $this->assertFalse(validate_password(true, true));
+            $this->assertFalse(validate_password(false, false));
+        }
+
+        public function testSendIntegerValues()
+        {
+            $this->assertFalse(validate_password(10, 10));
+        }
+
+        public function testSendZeroValues()
+        {
+            $this->assertFalse(validate_password(0, 0));
+        }
+}
+
+### EOF
```

Comment:
Here's a tip: to update a pull request, just push to the same branch of yours. If there's history conflict (e.g. when squashing commits), use --force when pushing.

On 2015 m. spalis 29 d. 12:53:41 EET, Alex Pandele <notifications@github.com> wrote:
>> +            $this->assertFalse(validate_password(true, true));
>> +            $this->assertFalse(validate_password(false, false));
>> +        }
>> +
>> +        public function testSendIntegerValues()
>> +        {
>> +            $this->assertFalse(validate_password(10, 10));
>> +        }
>> +
>> +        public function testSendZeroValues()
>> +        {
>> +            $this->assertFalse(validate_password(0, 0));
>> +        }
>> +}
>> +
>> +### EOF
>
>Sure, not an issue ! Regarding the pr, I just started using github and
>still trying to figure out how does it work
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/vexim/vexim2/pull/91/files#r43371349

-- 
Rimas